format/tmux/formater.go

```diff
@@ -17,7 +17,8 @@ type Config struct {
 	Symbols symbols
 	// Styles contains the tmux style strings for symbols and Git status
 	// components.
-	Styles styles
+	Styles  styles
+	Display display
```

Comment:
Can you add a godoc-friendly comment here.
Please see the differences between:

    go doc ./format/tmux Config.Symbols

and

    go doc ./format/tmux Config.Display